benchmark/runner/conf/benchmark/savina_micro_fjthrput.yaml

```diff
@@ -27,3 +27,10 @@ targets:
       actors: ["-D", "numWorkers=<value>"]
     run_args:
       messages: ["--numMessagesPerReactor", "<value>"]
+  lf-c:
+    copy_sources:
+      - "${lf_path}/benchmark/C/Savina/src/micro/fjthrput/Throughput.lf"
+    lf_file: "Throughput.lf"
+    binary: "Throughput"
+    gen_args:
+      messages: ["-D", "numWorkers=<value>"]
```

Comment:
This maps the `messages` parameter to the `numWorkers` cog variable, which is wrong. Actually, there should be two variables, one for the number of messages and one for the number of workers.